CONTRIBUTING.md

```diff
@@ -1,57 +1,108 @@
-# Contributing
+# Contributing to Backpack
+You want to help us enable Skyscanner to create beautiful, coherent products at scale? That's awesome! ❤️
 
-In this document we describe how to setup this repository for development and the release process.
+## Table of contents
 
-## Code style
+* [Prerequisites](#prerequisites)
+* [Getting started](#getting-started)
+* [Adding a new component](#adding-a-new-component)
+* [How we review Backpack contributions](#how-we-review-backpack-contributions)
+* [How to](#how-to)
 
-**All new components should be written in Swift.**
-
-Please follow the [New York Times Objective-C style-guide](https://github.com/NYTimes/objective-c-style-guide) when writing Objective-C. Follow other conventions and patterns established in the source code already when the style-guide cannot help you. The goal is that the codebase should look like it was written by a single author.
+## Prerequisites
 
-Wrap all Objective-C in `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` blocks. Make sure to annotate any types that are nullable correctly.
+### Licence
+By contributing your code, you agree to license your contribution under the terms of the [APLv2](./LICENSE).
 
-## How we review Backpack contributions
-
-Please see the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md).
+All files are released with the Apache 2.0 licence.
 
-## Environment
+### Environment
 
 We use both Ruby and Node in this project. To manage these language runtimes we recommend using [`rbenv`][0] and [`nvm`][1] respectively. You should ensure you have these installed or some other means of handling Ruby and Node versions. The required ruby version is specified in `.ruby-version` and the Node version is in `.nvmrc`.
 
 With `rbenv` use `rbenv install` to install the project's version of Ruby. For `nvm` use `nvm use`. Also ensure you install `bundler` for Ruby with `gem install bundler`.
 
 Use the most recent stable version of Xcode, however the project should work with old versions of the same major.
```

Comment:
Should we maybe link here to where we define it in the build/ci parts just so that people don't E.g. leap ahead whilst we are still working on making it compatible?